src/index.js

```diff
@@ -172,17 +323,63 @@ const renderLevel = (root, levelNumber) => {
 
     if (soko.goto(row, col)) {
       renderBoard();
+
+      const lastMove = soko.sequence().charAt(soko.sequence().length - 1);
+      const key = Object.keys(moves).find(key => moves[key] === lastMove);
+      rotatePlayer(key);
+
+      if (soko.solved()) {
+        const score = localStorage.getItem(levelNumber + 1);
+        if (score) {
+          if (score > soko.movesCount()){
+            updateBestScore(levelNumber + 1, soko.movesCount());
+          }
+        } 
+        else if (!score) {
+          updateBestScore(levelNumber + 1, soko.movesCount());
+        }
+        renderLevelComplete();
+      }
     }
   });
+
+ updateStatus = () => {
+    document.getElementById("level-label").innerHTML = `${soko.levelNumber() + 1}`;
+    document.getElementById("current").innerHTML = `${soko.movesCount()}`;
+    document.getElementById("best")
+      .innerHTML = `${(localStorage.getItem(soko.levelNumber() + 1) === null)  ? 
+      "&infin;" : (localStorage.getItem(soko.levelNumber() + 1))}`;
+    document.getElementById("sequence").innerHTML = `${soko.sequence()}`;
+  };
+
+  collapseEl.addEventListener("click", event => {
+    statsEl.classList.toggle("active");
```

Comment:
I don't think this does anything since the class doesn't seem to be in the stylesheet `.active`.